src/models/HydroReservoir.jl

```diff
@@ -0,0 +1,128 @@
+#=
+This file exists here because HydroTurbine depends on it. Since autogenerated
+models sort imports alphabetically, HydroTurbine always appeared first, causing
+an import error.
+=#
+
+#! format: off
+
+"""
+    mutable struct HydroReservoir <: Component
+        name::String
+        available::Bool
+        storage_volume_limits::Union{Nothing, MinMax, TimeSeriesKey}
+        spillate_outflow_limits::Union{Nothing, MinMax, TimeSeriesKey}
+        inflow::Union{Float64, TimeSeriesKey}
+        outflow::Union{Float64, MinMax, TimeSeriesKey}
+        volume_targets::Union{Float64, MinMax, TimeSeriesKey}
```

Comment:
```suggestion
        volume_targets::Union{MinMax, TimeSeriesKey}
```
Having a float and the same min/max is redundant